Jenkinsfile

```diff
@@ -381,6 +381,30 @@ pipeline {
       }
     }
 
+     stage('L2: Sparrowhawk Tests') {
+      when {
+        anyOf {
+          branch 'main'
+          changeRequest target: 'main'
+        }
+      }
+      failFast true
+      stages {
+        stage('L2: EN ITN Run Sparrowhawk test') {
+          steps {
+            sh 'CUDA_VISIBLE_DEVICES=""  cd tools/text_processing_deployment && bash sh_test.sh  --MODE=notest --OVERWRITE_CACHE=False --FAR_PATH=${EN_TN_CACHE}/SH_ITN --LANGUAGE="en"'
+
+          }
+        }
+        stage('L2: EN TN Run Sparrowhawk test') {
+          steps {
+            sh 'CUDA_VISIBLE_DEVICES=""  cd tools/text_processing_deployment && bash sh_test.sh --MODE=notest --OVERWRITE_CACHE=False --FAR_PATH=${EN_TN_CACHE}/SH_TN --GRAMMARS="tn_grammars" --LANGUAGE="en" '
```

Comment:
Why `--MODE=notest`? Do you want to just check export? How about integrating these tests tests/nemo_text_processing/en/test_sparrowhawk_normalization.sh ?

Reply:
Fixed